pom.xml

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
split into different lines.

Reply:
formate by vscode.

refer https://github.com/apache/dubbo/blob/3.0/pom.xml#L17
https://github.com/apache/kylin/blob/main/pom.xml#L19

Reply:
should keep the same  with nebula's all java project. refer https://github.com/neo4j/neo4j-java-driver/blob/3eb361338b930d26becb4c23e961661295c26a4e/pom.xml#L1